XeroNetStandardApp/Controllers/Accounting/ManualJournalInfoController.cs

```diff
@@ -1,198 +1,136 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Net.Http;
 using System.Threading.Tasks;
 using Xero.NetStandard.OAuth2.Api;
-using Xero.NetStandard.OAuth2.Client;
 using Xero.NetStandard.OAuth2.Config;
 using Xero.NetStandard.OAuth2.Model.Accounting;
-using Xero.NetStandard.OAuth2.Token;
 
 namespace XeroNetStandardApp.Controllers
 {
+    /// <summary>
+    /// Controller implementing methods demonstrating following accounting endpoints:
+    /// <para>- GET: /ManualJournalInfo/</para>
+    /// <para>- POST: /ManualJournalInfo#Create</para>
+    /// </summary>
     public class ManualJournalInfo : Controller
     {
-        private readonly ILogger<ManualJournalInfo> _logger;
-        private readonly IOptions<XeroConfiguration> XeroConfig;
-        private readonly IHttpClientFactory httpClientFactory;
+        private readonly IOptions<XeroConfiguration> _xeroConfig;
+        private readonly AccountingApi _accountingApi;
 
-        public ManualJournalInfo(IOptions<XeroConfiguration> XeroConfig, IHttpClientFactory httpClientFactory, ILogger<ManualJournalInfo> logger)
+        public ManualJournalInfo(IOptions<XeroConfiguration> xeroConfig)
         {
-            _logger = logger;
```

Comment:
@JRising-Xero has the logger been relocated within the app so it doesn't have to be implemented in every controller?

Reply:
Logger wasn't being used in the controller, so, I removed it. Keeping it just clutters the controller.
For some context the logger is injected into the entire project as apart of `CreateHostBuilder` method in `Program.cs`. You can easily add the logger to controllers using dependency injection (which is what was originally done). Users can always add it back into the controller via the constructor if need be.